src_c/image.c

```diff
@@ -1221,6 +1221,40 @@ image_frombytes(PyObject *self, PyObject *arg, PyObject *kwds)
     return (PyObject *)pgSurface_New(surf);
 }
 
+PyObject *
+image_tostring(PyObject *self, PyObject *arg, PyObject *kwarg)
+{
+    if (PyErr_WarnEx(PyExc_DeprecationWarning,
+                     "pygame.image.tostring deprecated since 2.3.0",
+                     1) == -1) {
+        return NULL;
+    }
+
+    PyObject *result = image_tobytes(self, arg, kwarg);
+    if (result == NULL) {
+        return NULL;
+    }
+
+    return result;
+}
+
+PyObject *
+image_fromstring(PyObject *self, PyObject *arg, PyObject *kwarg)
+{
+    if (PyErr_WarnEx(PyExc_DeprecationWarning,
+                     "pygame.image.fromstring deprecated since 2.3.0",
+                     1) == -1) {
+        return NULL;
+    }
+
+    PyObject *result = image_frombytes(self, arg, kwarg);
+    if (result == NULL) {
+        return NULL;
+    }
+
+    return result;
```

Comment:
```suggestion
    return image_frombytes(self, arg, kwarg);
```
super minor nitpick. You can also do the same for the above function